concourse/setup.py

```diff
@@ -273,6 +274,44 @@ def deploy_concourse_landscape(
     info('Setting teams on Concourse ...')
     set_teams(config=concourse_cfg)
 
+
+def destroy_concourse_landscape(config_name: str, release_name: str, helm_cmd_path: str):
```

Comment:
why not pass the cfg_set?
also, I would prefer to determine the helm path within this function (util.which('helm')); could be retained as an optional parameter (although I doubt this will add benefit)